pkg/injector/service/injector.go

```diff
@@ -56,12 +55,27 @@ var AllowedServiceAccountInfos = []string{
 	"tekton-pipelines:tekton-pipelines-controller",
 }
 
+type (
+	signDaprdCertificateFn func(context.Context, string) ([]byte, []byte, error)
+	currentTrustAnchorsFn  func() ([]byte, error)
+)
```

Comment:
Please, use named arguments and returned value, so we know what this function's expected and returned values are. Just seeing "string" or "[]byte" is not clear